spk/rutorrent/src/service-setup.sh

```diff
@@ -61,59 +64,65 @@ fix_shared_folders_rights()
     local folder=$1
     echo "Fixing shared folder rights for ${folder}"
 
```

Comment:
I'm assuming this only runs on DSM6?  For my own curiosity, can DSM7 installation method (e.g. with no direct intervention of file user/groups) be used as well for DSM6 (e.g. could it be possible to get rid of it entirely)?

Reply:
It seems that it's also running for DSM 7 as well and may still be required. Would need to check.